src/backend/tests/c-lisp/conditional.sexp

```diff
@@ -0,0 +1,15 @@
+(c-lisp
+    (define ((print int) (n int)))
+
+    (define ((main int))
+        (if
+            #t
+            ((call print 1)
+             (call print 2)))
+
+        (if
+            #f
+            ((call print 0))
+            else
```

Comment:
let's follow lisp syntax?

(if (cond) (pred) (else))

else block is optional.

Reply:
Sure. Now that I think of it, the Lisp syntax is it the more sensible option.

Reply:
Changed